src/GraphQL/Types/GraphType.cs

```diff
@@ -25,6 +25,11 @@ protected GraphType()
             }
         }
 
+        /// <inheritdoc/>
+        public virtual void Initialize(ISchema schema)
```

Comment:
Why so? I guess because of `ScalarGraphType`. If so I suggest to move `throw` here and override `ScalarGraphType.Initialize` to be empty.

Reply:
Done